frontendservice/src/router/index.js

```diff
@@ -80,6 +81,14 @@ const router = new VueRouter({
       component: Verify,
       meta: {
         requiresUnverified: true,
+      }
+    },
+    {
+      path: "/shop-owner/createshop",
```

Comment:
should we have "create-shop" or "createshop"?

Reply:
https://medium.com/@victor.leong.17/5-url-best-practices-936329ba36a7
Have a look here.

Reply:
Maybe our mentors can guide us here.